packages/connectivity/src/scp-cf/environment-accessor/service-bindings.ts

```diff
@@ -9,19 +9,19 @@ const logger = createLogger({
 
 /**
  * Services getter for a given service.
+ * Internally uses, xsenv library to read configurations from VCAP_SERVICES.
```

Comment:
[q] I chose to make all functions in this package public as I think they all could be used and don't see any behaviour change being introduced here until `xsenv` chnages something. WDY think?

Right now we only need `getServiceBinding` in the AI SDK.

Reply:
I wouldn't expose anything that we don't need. We used to do that a lot and block ourselves with this. Even though it is unlikely that things will change, it also unnecessarily clutters the API, making it bigger and less accessible.